2996_reflection/reflection.md

```diff
@@ -6302,7 +6308,9 @@ consteval info value_of(info r);
 
 [#]{.pnum} *Constant When*: `r` is a reflection representing
 
-* [#.#]{.pnum} either an object or variable, usable in constant expressions from some point in the evaluation context ([expr.const]), whose type is a structural type ([temp.type]),
+* [#.#]{.pnum} a value,
+* [#.#]{.pnum} an enumerator, or
```

Comment:
Value and enumerator are already covered in the two bullets after the object one. Did you mean to delete those two, so that the object/variable case is last?

Reply:
Yep, that's what I meant to do 😄 Thanks for the catch!